etc/ex.mock.config.yaml

```diff
@@ -17,3 +17,6 @@ openshift:
   target: {{OPENSHIFT_TARGET}}
   user: {{OPENSHIFT_USER}}
   pass: {{OPENSHIFT_PASS}}
+broker:
```

Comment:
I'd prefer to add these in until #171 is discussed and merged at a later date. These aren't used now, but if for whatever reason somebody looks at it, they should reflect the current schema.